prow/scripts/provision-vm-and-run-manager.sh

```diff
@@ -124,10 +127,10 @@ fi
 
 log::info "Copying Reconciler to the instance"
 #shellcheck disable=SC2088
-utils::compress_send_to_vm "${ZONE}" "keda-manager-test-${RANDOM_ID}" "/home/prow/go/src/github.com/kyma-project/keda-manager" "~/keda-manager"
+utils::compress_send_to_vm "${ZONE}" "${JOB_REPO_NAME}-test-${RANDOM_ID}" "/home/prow/go/src/github.com/kyma-project/${JOB_REPO_NAME}" "~/${JOB_REPO_NAME}"
 
 log::info "Triggering the installation"
 # TODO: Below line is a workaround -> Check issue https://github.com/kyma-project/test-infra/issues/6513
-utils::ssh_to_vm_with_script -z "${ZONE}" -n "keda-manager-test-${RANDOM_ID}" -c "sudo bash -c \"export PATH=\$PATH:\$HOME/keda-manager/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin && cd \$HOME/keda-manager && make -C hack/local run\""
-#&&	PATH=$PATH:$HOME/keda-manager/bin
-log::success "all done"
\ No newline at end of file
+utils::ssh_to_vm_with_script -z "${ZONE}" -n "${JOB_REPO_NAME}-test-${RANDOM_ID}" -c "sudo bash -c \"export PATH=\$PATH:\$HOME/${JOB_REPO_NAME}/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin && cd \$HOME/${JOB_REPO_NAME} && ${JOB_MAKE_TARGET}\""
```

Comment:
Could we make it able to customize the whole command in order to make it possible to use it with https://github.com/kyma-project/test-infra/blob/main/prow/scripts/provision-vm-and-test-serverless-manager.sh